src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.SupportedTargetFrameworks.props

```diff
@@ -28,6 +28,11 @@ Copyright (c) .NET Foundation. All rights reserved.
                                             Condition="$([MSBuild]::VersionGreaterThanOrEquals($(MSBuildVersion), '17.4.0'))"/>
     </ItemGroup>
 
+    <PropertyGroup>
+        <UnsupportedTargetFrameworkVersion>8.0</UnsupportedTargetFrameworkVersion>
+        <MinimumVisualStudioVersionForUnsupportedTargetFrameworkVersion>17.8</MinimumVisualStudioVersionForUnsupportedTargetFrameworkVersion>
```

Comment:
Is there a way we can know when these need bumping automatically? Or do we just add it to the 'things we bump when we version' list?

Reply:
I don't think there's a way to know for the VS version as that version can change with each major. I think we can expect 4 releases a year most of the time so it's probably predictable for a couple of years but would have to be updated with each major VS release. Might be safer to just update every year.

For the UnsupportedTFM, I might be able to calculate it from the NETCoreAppMaximumVersion value. Is it worth doing if we have to update the VS version each time?

Reply:
No, you make a good case for keeping it manual. I wasn't sure if we had the same data available elsewhere easily or not, that's all.